clientv2/client.go

```diff
@@ -338,25 +348,26 @@ type response struct {
 	Errors json.RawMessage `json:"errors"`
 }
 
-func unmarshal(data []byte, res interface{}) error {
+func (c *Client) unmarshal(data []byte, res interface{}) (err error) {
```

Comment:
I like the way it's written.
```suggestion
func (c *Client) unmarshal(data []byte, res interface{}) error {
```